pages/blog/2022-summary.md

```diff
@@ -0,0 +1,252 @@
+/>
+
+## Social media
+
+Social media are not a silver bullet. I think it should only be a news interface with the community, not a marketing tool.
+
+Below data will show you that these channels should not be treated as promotion channels that convert to new users. You will see that LinkedIn and Twitter brought together 2 082 users to the AsyncAPI website. Now, compare it to [this single documentation article from Mozilla](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) that referred to our website 5 542 users.
+
+Social media is suitable for communication with the community. Promotion should target people in a location where they look for help. In other words, mentioning AsyncAPI on a WebSocket-related website performs much better than hundreds of social media posts. So we should also try to make sure AsyncAPI is mentioned in other locations. Why not mention AsyncAPI in official docs for Kafka, MQTT, and other protocols?
+
+### LinkedIn
+
+I'm so damn tired of LinkedIn. Seriously, a platform that drains so much data from others and earns money from job marketing cannot afford and give users access to historical data and see more than one year-old stats. Like seriously...
+
+We had 2139 followers at the end of 2022. Every year's data since account creation looks like below.
+
+<Figure
+  src="/img/posts/2022-summary/linkedin-followers.webp"
+/>
+
+We are not active on LinkedIn. We only republish whatever we publish on Twitter. No engagement is created. We treat it as a news feed, although LinkedIn != Twitter.
+
+Numbers went down a lot compared to last year, but when you look two years back, numbers are very similar. Why? in 2021 we spent $1200 on LinkedIn marketing; thus the number spike for one year.
+
+<Figure
+  src="/img/posts/2022-summary/linkedin-referrals.webp"
+/>
+
+My statement doesn't change. LinkedIn is an important presence, and even without dedicated moderation, simple reposting brings people to the project. We should continue our work there.
+
+### Twitter
+
+Twitter? Seriously, the most unreliable tool ever :smiley: but of course, I'm biased; I do not like it. So let's look at the data. Are people really leaving Twitter because of Elon Musk?
+
+Forget about the number of followers. We have over 3.5k followers, over 500 more than last year. For me, followers number are as reliable as GitHub stars on a repository. Not much...
+
+Let's see how Twitter's presence converts to visits to the AsyncAPI website.
+
+<Figure
+  src="/img/posts/2022-summary/twitter-referrals.webp"
+/>
+
+Exactly the same situation as with LinkedIn. Numbers drop between 2021 and 2022, but again because 2021 was a special year. We experimented that year with paid marketing on Twitter too. So the numbers we should compare are for the years 2020 and 2022. As we can see no change, and it is pretty stable. It looks like Twitter was not really abandoned, at least not for now :smiley:.
```

Comment:
```suggestion
Exactly the same situation as with LinkedIn. Numbers drop between 2021 and 2022, but again because 2021 was a special year. We experimented that year with paid marketing on Twitter too. So the numbers we should compare are for the years 2020 and 2022. As we can see no change, and it is pretty stable. It looks like Twitter was not really abandoned, at least not for now :smiley: .
```